pkg/clusteragent/admission/mutate/auto_instrumentation.go

```diff
@@ -299,38 +344,27 @@ func extractLibrariesFromOwnerAnnotations(pod *corev1.Pod, registry string) []li
 func extractLibrariesFromAnnotations(
 	pod *corev1.Pod,
 	registry string,
-	libInfoMap map[string]libInfo,
 ) []libInfo {
 	annotations := pod.Annotations
 	libList := []libInfo{}
 	for _, lang := range supportedLanguages {
 		customLibAnnotation := strings.ToLower(fmt.Sprintf(customLibAnnotationKeyFormat, lang))
 		if image, found := annotations[customLibAnnotation]; found {
-			if li, ok := libInfoMap[string(lang)]; ok {
-				log.Debugf(
-					"Found %s library annotation %s, will overwrite %s injected with Single Step Instrumentation",
-					string(lang), image, li.image,
-				)
-			}
-			libInfoMap[string(lang)] = libInfo{
-				lang:  lang,
-				image: image,
-			}
+			log.Debugf(
+				"Found %s library annotation %s, will overwrite %s injected with Single Step Instrumentation",
+				string(lang), image, image,
```

Comment:
Should this be `string(lang), customLibAnnotation, image` instead?

Reply:
Good catch. I'm making a separate PR to fix custom image/container annotation injection as well.